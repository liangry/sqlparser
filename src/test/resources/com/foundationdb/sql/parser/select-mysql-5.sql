SELECT SQL_NO_CACHE a,b,c FROM t1
