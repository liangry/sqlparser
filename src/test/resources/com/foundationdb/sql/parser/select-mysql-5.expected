com.foundationdb.sql.parser.CursorNode@409c8a10
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	com.foundationdb.sql.parser.SelectNode@652c4193
	isDistinct: false
	cacheHint: false
	resultColumns: 		
		com.foundationdb.sql.parser.ResultColumnList@3c854dc9

		[0]:		
		com.foundationdb.sql.parser.ResultColumn@53b5e629
		exposedName: a
		name: a
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			com.foundationdb.sql.parser.ColumnReference@4b19b8ae
			columnName: a
			tableName: null
			type: null
		[1]:		
		com.foundationdb.sql.parser.ResultColumn@5abce07
		exposedName: b
		name: b
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			com.foundationdb.sql.parser.ColumnReference@52633079
			columnName: b
			tableName: null
			type: null
		[2]:		
		com.foundationdb.sql.parser.ResultColumn@6e038230
		exposedName: c
		name: c
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			com.foundationdb.sql.parser.ColumnReference@285855bd
			columnName: c
			tableName: null
			type: null
	fromList: 		
		com.foundationdb.sql.parser.FromList@3f64fffc

		[0]:		
		com.foundationdb.sql.parser.FromBaseTable@7fe219e7
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
